rtc/encryption.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Encryption information for the RTC protocol
+title_nav: RTC Encryption help
+description: Useful information for understanding how encryption is used with RTC
+keywords: rtc encrypt decrypt key rotate signature
+---
+
+This documentation is in progress. Please contact us with any suggestions you think should be here.
+
+> **Caution**: The advice on this page may not guarantee a secure connection. If data secrecy is important to you please consult a security professional.
+
+## Encryption keys are mandatory
+
+RTC requires an encryption key so {{site.companyname}} can prove to ourselves (and any interested customers) that the RTC server does not store document content in a format that we are able to read. We are very focussed on privacy and end-to-end encryption for this feature.
+
+## The difference between content encryption and JWT signing
+
+Encryption is a two way process (encryption and decryption) that transforms data into an encrypted form. Modern encryption algorithms produce an output that is indistinguishable from random data without the decryption key. RTC performs symmetric encryption (meaning encryption and decryption use the same key) on all document content, but the key is never sent to the server. As the key only exists on the client endpoints, the term for this is end-to-end encryption.
+
+Signing is a one way process where the data is not modified and can still be read by anyone. A signature is transmitted along with the data and can be used to validate the data has not been modified since it was approved by whoever has the signing key. RTC requires an [asymmetric signing algorithm]({{site.baseurl}}/rtc/jwt-authentication/#supportedalgorithms) for JWT identity tokens. Asymmetric signing uses a public/private key pair: signatures are created with the private key and verified using the public key. The public key stored on the Tiny Cloud servers can only be used to verify signatures, not create new ones, so it is quite safe for us to hold or cache without security implications should it be disclosed.
```

Comment:
```suggestion
JWT signing does not modify the data, and the data can be read by anyone (the data is not encrypted). A JWT signature is transmitted along with the data and is used to validate that the data has not been modified since the signature (JWT) was created. RTC requires an [asymmetric signing algorithm]({{site.baseurl}}/rtc/jwt-authentication/#supportedalgorithms) for JWT identity tokens. Asymmetric signing uses a public/private key pair: signatures are created with the private key and verified using the public key. The public key stored on the {{site.cloudname}} servers can only be used to verify signatures, not create new ones. This allows the {{site.cloudname}} server to verify that the data has not been tampered between the end-points, but does not allow {{site.companyname}} to decrypt the data.
```

Reply:
I was deliberately trying to describe signing in general, the process is not unique to JWT, but I guess for anyone unfamiliar with it the difference is meaningless.